README-PNL.md

```diff
@@ -54,19 +54,19 @@ into the database, instantiate DPdash, and run it.
 
 ### Define
 
-Define two variables `state` and `data` that can be used respectively for DPdash application data 
-and project data:
+Define two variables `state` and `data` that can be used respectively for DPdash application
+data and project data, as well as a third variable `DPDASH_IMG` that stores the path of the dpdash.sif image file::
 
     export state=/where/you/want/to/save/app/data
     export data=/where/you/have/PHOENIX/format/project/data
+    export DPDASH_IMG=/path/to/dpdash.sif
 
 
 `${state}` is an empty directory that will be populated with files once you do `./init.sh` but you need to have 
 [PHOENIX style directory trees](http://docs.neuroinfo.org/dpdash/en/latest/quick_start.html#create-data-persistence-directories) inside `${data}`.
 
```

Comment:
Bring the new direction to a separate paragraph here.

Reply:
Something like this? 3cb7ea2aea77c971cf2cb6f49460997e0fee0989